packages/trpc/server/routers/viewer/bookings/get.handler.ts

```diff
@@ -351,6 +352,28 @@ export async function getBookings({
       take: take + 1,
       skip,
     }),
+    prisma.booking.findMany({
+      where: {
+        eventType: {
+          parent: {
+            team: {
+              members: {
+                some: {
+                  userId: user.id,
+                  role: {
+                    in: ["ADMIN", "OWNER"],
+                  },
```

Comment:
I see this query repeated at multiple places, I think it could be a good idea to query it separately(before all these queries) then simply do members: {
 id: {in: membershipIdsWhereUserIsAdminOwner}
}

Reply:
Oh yeah, that's a good point!